### DISCUSSION TITLE
Unable to authenticate wallet in unity.

### DISCUSSION THREAD

**Discussion by Sushmita1301** [WalletConnect/walletconnect-monorepo#2440] [Developer Support]
1.After confirming my wallet authorization in metamask app, the wallet address is not taken by the app. This step fails to authenticate the wallet for the latest 6.5.0 metamask app.
2.It seems to work fine for the older version of metamask app (v6.3.0)
3.I have used all the alphanumerical subdomains of the bridge url such as "f.bridge.walletconnect.org", "m.bridge.walletconnect.org", "8.bridge.walletconnect.org", etc.
It doesn't seem to work on android either for me. Are there any updates or suggestions to where might the problem occur and how I can fix it?

**Comment by finessevanes** [comment]
Hey - what SDK and version are you using? It looks like you're trying to use the v1 bridge, which we are sunsetting in less than a month.

**Comment by Sushmita1301** [comment]
Could you let me know the v2 relay bridge please?

**Reply by finessevanes** [reply]
wss://relay.walletconnect.com
